GitUI/Translation/English.xlf

```diff
@@ -1558,6 +1558,10 @@ The primary difftool can still be selected by clicking the main menu entry.</sou
         <source>&amp;Decrease the number of lines of context</source>
         <target />
       </trans-unit>
+      <trans-unit id="diffAppearanceToolStripMenuItem.Text">
+        <source>&amp;Diff appearance</source>
```

Comment:
Side question for my personal knowledge: How do you use that in this contextual menu? Is it really useful to put access keys in this menu?

Because when I press "alt" to use it, the menu is immediately closed.

Reply:
Shift-F10

Reply:
> Because when I press "alt" to use it, the menu is immediately closed.

It is (context menu key or Shift-F10 or right click), then just the letter.

Reply:
> > Because when I press "alt" to use it, the menu is immediately closed.
> 
> It is (context menu key or Shift-F10 or right click), then just the letter.

In fact, that is not the access key that is used but the 1st letter of menu item label (and if there are more than one with the same letter, it circle between each ones).

That's why I was wondering if you have extra knowledge and that I think that access keys are even misleading...and should be removed?

Reply:
Forgot what I said. You're right.

Reply:
Msft thinks user may be confused by underlined characters.
Just turn on
![image](https://github.com/gitextensions/gitextensions/assets/36601201/7a5c9848-ec6a-4a3c-b05f-ac6d0eed88a0)